Tribler/Test/test_gui_dialogs.py

```diff
@@ -176,18 +175,26 @@ def disable_dispersy():
         self.startTest(disable_dispersy)
 
     def test_confirmationdialog(self):
```

Comment:
@whirm looks like a race condition of some sort. I can't get it reproduced on my ubuntu, 5 runs and all succeed. https://jenkins.tribler.org/job/Test_tribler_laurens2/23/console also succeeds on bbq. So it looks like a race..